app/services/github/repositories/sync.rb

```diff
@@ -13,15 +13,12 @@ def call
         Rails.logger.info("[GH] -- Repositories: #{repositories.count}")
 
         repositories.map do |repository|
-          repository_owner, repository_name = repository.link.split('/')[-2..-1]
+          languages = fetch_repository_languages(repository.link)
```

Comment:
I can see in [a comment below](https://github.com/Codeminer42/Punchclock/pull/479/files#diff-709bdfbc5d5e62f5efeb7dc03818d40bbab96c792086f334ee5a2917dff7f4b4R37) that this can be `nil`.

If `languages` is nil, should we really update the repository `language`?

Reply:
I don't get it. I didn't see any change or reply addressing this question :thinking: .